trantor/utils/AsyncFileLogger.cc

```diff
@@ -184,7 +186,17 @@ AsyncFileLogger::LoggerFile::LoggerFile(const std::string &filePath,
 #ifndef _MSC_VER
     fp_ = fopen(fileFullName_.c_str(), "a");
 #else
-    fp_ = _fsopen(fileFullName_.c_str(), "a+", _SH_DENYWR);
+    // Convert UTF-8 file to UCS-2
+    int nSizeNeeded = ::MultiByteToWideChar(
+        CP_UTF8, 0, &fileFullName_[0], (int)fileFullName_.size(), NULL, 0);
+    std::wstring wFullName(nSizeNeeded, 0);
+    ::MultiByteToWideChar(CP_UTF8,
+                          0,
+                          &fileFullName_[0],
+                          (int)fileFullName_.size(),
+                          &wFullName[0],
+                          nSizeNeeded);
```

Comment:
Please pack UTF-8 to UCS-2 (and the reverse) conversions into `trantor::utils::internal` to make future maintenance easier. (in `internal` because this is Windows only), Feel free to pull trantor in drogon and then alias the implementation.